src/net/sourceforge/kolmafia/textui/command/MirrorLogCommand.java

```diff
@@ -20,8 +20,8 @@ public void run(final String command, String parameters) {
       RequestLogger.closeMirror();
       KoLmafia.updateDisplay("Mirror stream closed.");
     } else {
-      if (!parameters.endsWith(".txt")) {
-        parameters += ".txt";
+      if (!parameters.endsWith(".txt") && !parameters.endsWith(".html")) {
+        parameters += ".html";
```

Comment:
Keep it as a txt

Reply:
Mind explaining why? The log is basically written as html and allowing it to be appended with the .html tag means it can be easily viewed as such.

Reply:
The tools I use to process mirror files will select the files of interest based upon a .txt ext.  This is all about whose workflow has to be changed and I am quite willing to oppose making me change mine trivial as the adjustments may be :-)

Reply:
Does the new change of the default being .txt satisfy?

Reply:
The file produced will not be a valid HTML file

Reply:
It wouldn't be no, but most modern browsers will still render it correctly.

Reply:
I feel like this is resolved now, we're making it txt unless explicitly set to html